src/main/java/com/codahale/metrics/newrelic/NewRelicReporter.java

```diff
@@ -142,6 +143,12 @@ public void report(
     timeTracker.tick();
   }
 
+  @Override
+  public void close() {
+    super.close();
+    sender.shutdown();
```

Comment:
I don't know how much it matters, but how did you choose which order to call these in?

Reply:
I agree that it probably doesn't matter...but I assumed that you'd want to do parent/default cleanup first before doing the implementation-specific cleanup.  🤷